docs/services/blockstorage.rst

```diff
@@ -1,89 +1,187 @@
 =============
 
-Commands for Rackspace Cloud Block Storage.
+This section lists all block storage commands supported by Rackspace CLI.
 
-Synopsis
+Commands
 --------
 
-::
+All block storage commands are based on this syntax::
 
    rack block-storage <resource> <action> [command flags]
 
-Commands
---------
+*Command flags* allow you to customize certain attributes of the command,
+such as ``--name`` to name a volume. Type ``rack block-storage <resourse> <action> --help``
+to bring up a list *command flags* specific to the command.
 
 ``volume``
 ~~~~~~~~~~
 
-Block Storage Volume operations
+Block storage volume commands use this syntax::
+
+    rack block-storage volume <action> [optional flags]
 
 ``list``
 ^^^^^^^^
-Usage::
+Retrieves a list of volumes::
 
     rack block-storage volume list [optional flags]
+    
+**Response**
+
+.. code::
+   
+    $ rack block-storage volume list
+    ID					                          Name	 Size	 Status		    Description	VolumeType	SnapshotID	Attachments														Created
+    81c7a7e5-01a5-44bb-9b43-0cc9f7c4e423		     75	   available		            SATA				[]															<nil>
+    a7c97db7-a0d3-495b-9ba3-1cb9dd9cf12c	Store	 75	   in-use			            	SATA			[map[host_name:<nil> device:/dev/xvdb server_id:8a254ea3-77b5-4f74-a893-8d2d51ae2cca id:a7c97db7-a0d3-495b-9ba3-1cb9dd9cf12c volume_id:a7c97db7-a0d3-495b-9ba3-1cb9dd9cf12c]]	<nil>
+    ca50fdfd-21f2-47e9-8ede-1518cb7467af	Store  150	 in-use				            SSD				[map[host_name:<nil> device:/dev/xvda server_id:8a254ea3-77b5-4f74-a893-8d2d51ae2cca id:ca50fdfd-21f2-47e9-8ede-1518cb7467af volume_id:ca50fdfd-21f2-47e9-8ede-1518cb7467af]]	<nil>
 
 ``create``
 ^^^^^^^^^^
-Usage::
+Creates a volume::
 
     rack block-storage volume create --size <volumeSize> [optional flags]
+    
+**Response**
+
+.. code::
+    
+    $ rack block-storage volume create --size 75 --name Response
+    ID		66dcbe53-1b62-4a15-adc2-e46e78b95f8b
+    Name		Response
+    Description
+    Size		75
+    VolumeType	SATA
+    SnapshotID
+    Attachments	[]
+    CreatedAt	2015-08-05T19:48:28.000000
 
 ``get``
 ^^^^^^^
-Usage::
+Retrieves details on a specified volume::
 
     rack block-storage volume get --id <volumeID> [optional flags]
     rack block-storage volume get --name <volumeName> [optional flags]
     (echo volumeID1 && echo volumeID2) | rack block-storage volume get --stdin id [optional flags]
 
+**Response**
+
+.. code::
+    
+    $ rack block-storage volume get --name Response
+    ID		66dcbe53-1b62-4a15-adc2-e46e78b95f8b
+    Name		Response
+    Description
+    Size		75
+    VolumeType	SATA
+    SnapshotID
+    Attachments	[]
+    CreatedAt	2015-08-05T19:48:28.000000
+
+
 ``update``
 ^^^^^^^^^^
-Usage::
+Updates the name and description of a volume::
 
     rack block-storage volume update --id <volumeID> [optional flags]
     rack block-storage volume update --name <volumeName> [optional flags]
 
+
 ``delete``
 ^^^^^^^^^^
-Usage::
+Permanently removes a volume::
```

Comment:
s/removes/deletes

'Removes' might be interpreted as detaching the volume from a server.